python/semantic_kernel/connectors/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -96,7 +96,12 @@ def __init__(
 
         # If the api_key is none, and the ad_token is none, and the ad_token_provider is none,
         # then we will attempt to get the ad_token using the default endpoint specified in the Azure OpenAI settings.
-        if api_key is None and ad_token_provider is None and azure_openai_settings.token_endpoint and ad_token is None:
+        if (
+            azure_openai_settings.api_key is None
+            and ad_token_provider is None
```

Comment:
can these also be supplied through env?

Reply:
Not at this moment. They're concrete objects.